components/row-events.js

```diff
@@ -20,10 +20,13 @@ const RowEvent = props => {
         </span>
         <span className="card_icons">
           <Icon name="users" />
-          {`${props.yesCount} attended`}
+          {props.yesCount}
+          {moment(props.time).isAfter() ? ' attending' : ' attended'}
```

Comment:
Thats a very clever way of doing but in the object of props we are receiving `status` key,
![image](https://user-images.githubusercontent.com/17708702/31703139-b777633c-b3f8-11e7-8611-d76fd121978b.png)
Do you mind checking that props and getting `attending or attended` terminology

Reply:
Ah I see! So something like...

```javascript
props.status === 'upcoming' ? ' attending' : ' attended'
```
...would be more ideal? I'll commit and push when I've time tomorrow. Thanks for the fast reply

Reply:
yes thank for helping out. Also for faster processing for PR, if it is major feature change or something, We encourage people to deploy it to Now.sh and share the link.
- install now-cli `npm i -g now` and run `now` inside that directory and follow instructions to login as this first time
- thats it just one command and app is deployed
- after app is deployed you can alias your now.sh deployment to PR number like coderplex-pr-11.now.sh and for every changes run `now` and after completion run `now alias <url> coderplex-pr-11.now.sh` to update it

Reply:
coderplex-app-pr20.now.sh/events

Reply:
i think this matter can go into Contributing docs